.pre-commit-config.yaml

```diff
@@ -1,4 +1,4 @@
-exclude: '^databricks_src|^data_engineering/'
+exclude: "^databricks_src|^data_engineering/"
```

Comment:
I believe the changes to this file are simply cosmetic, @ZoranPandovski can you confirm that this will not be a problem?